scripts/git-secrets-check.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+
+if [ -d /tmp/git-secrets ]; then
```

Comment:
nit: not critical but for consistency purposes these `-check` scripts usually just handle making sure the command/package is available and not performing the actual operation to run the command. This is mostly relevant to tools that will be used in multiple steps however.

Reply:
> nit: not critical but for consistency purposes these `-check` scripts usually just handle making sure the command/package is available and not performing the actual operation to run the command. This is mostly relevant to tools that will be used in multiple steps however.

@jtwillis92 this seems worthy of documenting somewhere for future reference 🗒️